contract.py

```diff
@@ -12,11 +12,13 @@ def wrb(w3, config_file):
 
 # Return the pricefeed contract, given a Ethereum address from "config.toml"
 def pricefeed(w3, config_file):
-  contract_address = config_file["contract"]["pricefeed"]
+  contract_addresses = config_file["contract"]["pricefeeds"]
+  contracts = []
     # ABI generated using
     # https://remix.ethereum.org
   with open("pricefeed_abi.json") as json_file:
     contract_abi = json.load(json_file)
-    contract = w3.eth.contract(contract_address, abi=contract_abi)
-    return contract
+    for address in contract_addresses:
+      contracts.append(w3.eth.contract(address, abi=contract_abi))
+    return contracts
```

Comment:
The former pythonista inside of me is saying:
```suggestion
    return [ w3.eth.contract(address, abi=contract_abi) for address in contract_addresses ]
```